tests/callbacks/test_device_stats_monitor.py

```diff
@@ -85,6 +102,128 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) ->
     trainer.fit(model)
 
 
+@RunIf(min_gpus=1)
+def test_device_stats_gpu_from_nvidia_and_cpu(tmpdir):
```

Comment:
```suggestion
def test_device_stats_gpu_from_torch_and_cpu(tmpdir):
```
can we merge and parameterize with `cpu_tests`. This test and `test_device_stats_gpu_from_nvidia_no_cpu`. They are quite similar I think.